app/src/main/python/gogdl/launch.py

```
@@ -0,0 +1,284 @@
+                    reap_children()
+                    time.sleep(0.1)
+            while is_alive():
+                reap_children()
+                time.sleep(0.1)
+            reap_children()
+        except NoMoreChildren:
+            print("All processes exited")
+
+
+    else:
+        process = subprocess.Popen(command, env=environment, 
```

Ruff (0.12.2)

`subprocess` call: check for execution of untrusted input

(S603)